projects/plugins/crm/.gitattributes

```diff
@@ -1,24 +1,7 @@
-# Files not needed to be distributed in the package.
-.gitattributes       export-ignore
-.github/             export-ignore
-package.json         export-ignore
-gulpfile.esm.js      export-ignore
-.env                 export-ignore
-.env.testing         export-ignore
-codeception.yml      export-ignore
-codeception.dist.yml export-ignore
-.wp-env.json         export-ignore
-/.wordpress-org/**/* export-ignore
-/.wordpress-org      export-ignore
-.wordpress-org/**/*  export-ignore
-.wordpress-org       export-ignore
-/node_modules/**/*   export-ignore
-/node_modules/*      export-ignore
-node_modules         export-ignore
-
 # Files to include in the mirror repo, but excluded via gitignore
 # Remember to end all directories with `/**` to properly tag every file.
 # /src/js/example.min.js  production-include
+build/**                                        production-include
```

Comment:
Is this something we use at the moment?

Reply:
It will be. It was added as part of this PR - https://github.com/Automattic/jetpack/pull/29062 - and will be included in local development and production builds (but not the monorepo). It just looks like it was never added to `production-include` in `gitattributes` at the time.